static/templates/navbar_alerts/demo_organization_deadline.hbs

```diff
@@ -0,0 +1,5 @@
+<div data-step="1">
+    {{#tr}}
+    This is a demo organization and will be automatically deleted in {demo_org_days_remaining} days.
```

Comment:
The word "demo organization" should link to the /help/ documentation.

Reply:
Done :)